src/test/components/ListOfPublishedProfiles.test.js

```diff
@@ -139,8 +139,6 @@ describe('ListOfPublishedProfiles', () => {
       </Provider>
     );
 
-    const { container } = renderResult;
```

Comment:
Because I looked at this file, I took the opportunity to remove some uses of `container` and add uses of `screen`.